app/forms/flood_risk_engine/steps/local_authority_form.rb

```diff
@@ -0,0 +1,47 @@
+module FloodRiskEngine
+  module Steps
+    class LocalAuthorityForm < FloodRiskEngine::Steps::BaseForm
+
+      def self.factory(enrollment)
+        organisation = enrollment.organisation || Organisation.new
+        new(organisation, enrollment)
+      end
+
+      def self.locale_key
+        "flood_risk_engine.enrollments.steps.local_authority"
+      end
+
+      def self.name_max_length
+        200
+      end
+
+      validation :name_present? do
+        validates :name, presence: {
+          message: I18n.t("#{LocalAuthorityForm.locale_key}.errors.name.blank")
+        }
+      end
+
+      include ActiveModel::Validations
+      validation :email_format, if: :name_present? do
```

Comment:
yes pulls in custom validators ... I wasn't sure whether to put that in base form ?

I'm not sure what allow_nil: true  means ?  the scheme as it stands uses reform validation groups to ensure the format validator is not run iif the name is blank, so the errors reported to user are quite nice and specific